pkgs/applications/video/imagination/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchurl, autoreconfHook, docbook_xml_xslt, ffmpeg-full, glib
+, gnome, gtk3, intltool, libtool, libxslt, pkg-config, sox, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "imagination";
+  version = "3.6";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "139dgb9vfr2q7bxvjskykdz526xxwrn0bh463ir8m2p7rx5a3pw5";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    docbook_xml_xslt
+    intltool
+    libtool
+    libxslt
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  propagatedNativeBuildInputs = [ ffmpeg-full sox ];
+
+  buildInputs = [ glib gtk3 ];
+
+  preBuild = ''
+    substituteInPlace src/main-window.c --replace \
+      'gtk_icon_theme_load_icon(icon_theme,"image", 20, 0, NULL)' \
+      'gtk_icon_theme_load_icon(icon_theme,"insert-image", 20, 0, NULL)'
+    substituteInPlace src/main-window.c --replace \
+      'gtk_icon_theme_load_icon(icon_theme,"sound", 20, 0, NULL)' \
+      'gtk_icon_theme_load_icon(icon_theme,"audio-x-generic", 20, 0, NULL)'
```

Comment:
If I don't do this these two icons are blank in the UI. It seems like current GTK no longer has `image` and `sound`? Is this the right fix?

Reply:
Try adding `gnome.adwaita-icon-theme` to `buildInputs` instead.

Reply:
I did try that, but it doesn't fix it.